### PR TITLE
Add help block to cron job field

### PR DESCRIPTION
**Pull request by mlunoe**
---
~~⚠️ Is dependent on this reactjs-components feature: https://github.com/mesosphere/reactjs-components/pull/335~~ Merged. This PR is now updated with the new reactjs-components version 0.15.0-beta.1

---

This PR adds a help block for the cron input field

### CODE REVIEW

**Comment by orlandohohmeier** [comment]
There seems to be an issue with the reactjs components - tested against linked https://github.com/mesosphere/reactjs-components/pull/335/commits/9b165a23c4e923f5d68b165a85d605fb8a5dbf9f

> (index):17 Uncaught ReferenceError: props is not defined
  -  getHelpBlock	@ FieldInput.js?8585:187
  -  rende @ FieldInput.js?8585:288

**Comment by mlunoe** [comment]
Should be fixed now! Sorry

**Comment by mlunoe** [comment]
Ping: @jfurrow @orlandohohmeier

File: src/js/schemas/job-schema/Schedule.js [hunk 1]
```diff
@@ -35,7 +35,11 @@ const Schedule = {
     },
     timezone: {
       title: 'Time Zone',
-      description: 'Enter time zone in TZ format, e.g. America/New_York',
+      description: (
+        <span>
+          Enter time zone in <a href="http://www.timezoneconverter.com/cgi-bin/zonehelp" target="_blank">TZ format</a>, e.g. America/New_York.
```
Comment (jfurrow) [inline src/js/schemas/job-schema/Schedule.js]: It makes me a little uneasy to link to a third party website, but it is a helpful resource!

Comment (jfurrow) [inline src/js/schemas/job-schema/Schedule.js]: Actually, is this resource even helpful for this? I don't see TZ format described here.

Comment (mlunoe) [inline src/js/schemas/job-schema/Schedule.js]: I went over with @sascala, and we decided to link to this as Wikipedia was not ideal. If you click on the country you get the TZ format.
I agree with you that this is not ideal either, but in any case helpful

Comment (jfurrow) [inline src/js/schemas/job-schema/Schedule.js]: Ahh okay cool, thanks for the explanation!

**Comment by orlandohohmeier** [comment]
I still get the `ReferenceError`...

**Comment by mlunoe** [comment]
I am not sure how you get that. I updated this PR with the new release of reactjs-components containing the feature this PR dependent on, and it works for me.

**Comment by orlandohohmeier** [comment]
Now it's working! LGTM 🎆 🚢 🇮🇹

**Pull request merged**